x-pack/plugins/ml/public/application/settings/calendars/list/header.tsx

```diff
@@ -0,0 +1,137 @@
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*
+ * React component for the header section of the calendars list page.
+ */
+
+import type { FC } from 'react';
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import {
+  EuiSpacer,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiText,
+  EuiTextColor,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+
+import { MlPageHeader } from '../../../components/page_header';
+import { useMlKibana } from '../../../contexts/kibana/kibana_context';
+
+interface Props {
+  isDst: boolean;
+  totalCount: number;
+  refreshCalendars: () => void;
+}
+
+export const CalendarsListHeader: FC<Props> = ({ totalCount, refreshCalendars, isDst }) => {
+  const {
+    services: {
+      docLinks: { links },
+    },
+  } = useMlKibana();
+  const docsUrl = links.ml.calendars;
+  return (
+    <>
+      <MlPageHeader>
+        {isDst ? (
+          <FormattedMessage
+            id="xpack.ml.settings.calendars.listHeader.calendarsTitle"
+            defaultMessage="Daylight savings calendars"
+          />
+        ) : (
+          <FormattedMessage
+            id="xpack.ml.settings.calendars.listHeader.calendarsTitle"
+            defaultMessage="Calendars"
+          />
+        )}
+      </MlPageHeader>
+      <EuiFlexGroup justifyContent="spaceBetween" alignItems="baseline">
+        <EuiFlexItem grow={false}>
+          <EuiFlexGroup alignItems="baseline" gutterSize="m" responsive={false}>
+            <EuiFlexItem grow={false}>
+              <EuiTextColor color="subdued">
+                <p>
+                  <FormattedMessage
+                    id="xpack.ml.settings.calendars.listHeader.calendarsListTotalCount"
+                    defaultMessage="{totalCount} in total"
+                    values={{
+                      totalCount,
+                    }}
+                  />
+                </p>
+              </EuiTextColor>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiFlexGroup alignItems="baseline" gutterSize="m" responsive={false}>
+            <EuiFlexItem grow={false}>
+              <EuiButtonEmpty
+                size="s"
+                iconType="refresh"
+                onClick={refreshCalendars}
+                data-test-subj="mlCalendarListRefreshButton"
+              >
+                <FormattedMessage
+                  id="xpack.ml.settings.calendars.listHeader.refreshButtonLabel"
+                  defaultMessage="Refresh"
+                />
+              </EuiButtonEmpty>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiSpacer size="m" />
+      <EuiText>
+        <p>
+          <EuiTextColor color="subdued">
+            {isDst ? (
+              <FormattedMessage
+                id="xpack.ml.settings.calendars.listHeader.calendarsDescription"
```

Comment:
Should the id here contain `Dst`, e.g `calendarsDstDescription`

Reply:
Updated in https://github.com/elastic/kibana/pull/193605/commits/db72ef0440d4e606a284040d561af0f981647fc5